### PR TITLE
bug/color issues

### PR DESCRIPTION
**Pull request by Gnzlt**
## What does this implement/fix? Explain your changes.
- [T1500728](https://testrail.systems.mega.nz/index.php?/tests/view/1500728)
- [T1500732](https://testrail.systems.mega.nz/index.php?/tests/view/1500732)

## Any relevant logs, error output, etc?
...

## Where has this been tested?

- **Devices/Simulators:** 
Google Pixel 4

- **OS Version:** 
Android 11

### CODE REVIEW

**Review by javiergm1983 — APPROVED** [review]
Reviewed and tested. The reported issues have been fixed.

File: documentscanner/src/main/res/color-night/chip_text.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:color="#121212" android:state_checked="true" />
+    <item android:color="@color/dark_grey" android:state_checked="true" />
     <item android:color="#888A89" />
```
Comment (ash-mega) [inline documentscanner/src/main/res/color-night/chip_text.xml:4]: If this should use a defined colour reference instead?

**Review by ash-mega — COMMENTED** [review]
Review and tested, the changes are fine.
Only one minor question: there's still a place where doesn't use colour reference. Please see the code comment, cheers!

File: documentscanner/src/main/res/color-night/chip_text.xml (hunk 1 above)
Comment (javiergm1983) [reply documentscanner/src/main/res/color-night/chip_text.xml:4]: I think so. @Gnzlt please could you take a look?

**Comment by Gnzlt** [comment]
@ash-mega You were right, It's ready for review now 👍

**Review by ash-mega — APPROVED** [review]
Thank you, all good!
Approved.

**Review by javiergm1983 — APPROVED** [review]

**Pull request merged**